### ISSUE TITLE
Time for a rewrite?

### ISSUE THREAD

**Issue by Hypercubed** [heff/chg#10]
Not sure how much this is used but maybe time for a rewrite.  If acceptable I would like to:

* Update deps
* Dump support for old node versions
* Remove grunt and the grunt tasks
* Use es6 or TS.

Thoughts/objections?

**Comment by heff** [comment]
I have no objections to that.

@gkatsev is this still used for anything video.js related? And if so any thoughts?

**Comment by gkatsev** [comment]
I don't think we really use it anywhere in Video.js anymore. If this is still useful to people, I don't have objections to updating it and what not, though, I would prefer to avoid typescript.

**Comment by Hypercubed** [comment]
I may be the only one to use it!  I'll do an update if and when I have time... I'll leave it CJS.

**Comment by gkatsev** [comment]
We moved to conventional changelog for videojs.

Feel free to upgrade to es6 and es modules. And if you really prefer TS, don't let me stop you since I'm no longer using this :slightly_smiling_face:
